common/services/certificate.js

```diff
@@ -42,22 +58,26 @@ const verify2FA = function(code, priKey) {
 
 export default {
   getCertificate (ephemeralKey) {
-    return new Promise (function(resolve, reject) {
-      if (retrieveSavedCert() == null && retrieveSavedCert() == undefined) {
-        registerCertificate(ephemeralKey.publicKey).then(response => {
+    return new Promise (async function(resolve, reject) {
+      const cert = await retrieveSavedCert()
+      if (cert == null && cert == undefined) {
```

Comment:
Shouldn't this be an `or`?

Reply:
mmmmmmmmmmmm yes it sounds right, but that was the original code.

I'm not going to underestimate javascript devs but I think… they might be right 🤣 

<img width="257" alt="image" src="https://github.com/doordeck/doordeck-sdk-javascript/assets/6842786/ef09b03c-16d6-48f0-a798-16b9997b0267">

Reply:
we can keep the jokes ongoing...
<img width="201" alt="image" src="https://github.com/doordeck/doordeck-sdk-javascript/assets/6842786/ddd9be78-a721-40c7-8fb3-5b9732964f66">

Reply:
<img width="186" alt="image" src="https://github.com/doordeck/doordeck-sdk-javascript/assets/6842786/9962ad3b-14ff-40d3-ba92-aba539cb5bda">